syntax = "proto3";

service SearchService {
    rpc Search(SearchRequest) return (SearchResponse){}
}

message SearchRequest {
    string query = 1;
}

message SearchResponse {
    repeated SearchResult results = 1;
}

message SearchResult {
    string title = 1;
    string snipt = 2;
}